src/main.cpp

```diff
@@ -46,6 +47,30 @@
 #include "gesturefilterarea.h"
 #include "notificationsnoozer.h"
 
+// Add AppLauncher class
+class AppLauncher : public QObject
```

Comment:
It's probably nicer to move this to a dedicated source file. Maybe use `applaunchhelper` but rename it to `applauncher`